Connectivity Certs Controller chart (#2803)
kyma-project/kyma: resources/application-connector/charts/connectivity-certs-controller/README.md

# Connectivity Certs Controller

## Overview

> The Connectivity Certs Controller is responsible for fetching client certificate and root CA from Connector Service.

Reviewer: You could've used the description from the component readme.

> The Connectivity Certs Controller is responsible for fetching client certificate and root CA from Connector Service.

Reviewer: ```suggestion
The Connectivity Certs Controller fetches the client certificate and the root CA from the central Connector Service and saves them to Secrets.
```

## Configuration

> The Connectivity Certs Controller has the following parameters, that can be set through the chart:

Reviewer: ```suggestion
You can set the following parameters in the Connectivity Certs Controller chart:
```

> - **appName** - Name used in controller registration. The default value is `connectivity-certs-controller`.

Reviewer: Name of what?

> - **namespace** - Namespace in which secrets are created. The default is `kyma-integration`.

Reviewer: ```suggestion
- **namespace** - Namespace in which the Controller creates the Secrets that store certificates. The default Namespace is `kyma-integration`.
```

> - **clusterCertificatesSecret** - Secret name where cluster client certificate and key are kept. The default is `cluster-client-certificates`.

Reviewer: ```suggestion
- **clusterCertificatesSecret** - Name of the Secret which stores where the client certificate and key are. The default name is `cluster-client-certificates`.
```

> - **clusterCertificatesSecret** - Name of the Secret which stores where the client certificate and key are. The default name is `cluster-client-certificates`.

Reviewer: ```suggestion
- **clusterCertificatesSecret** - Name of the Secret which stores the client certificate and key. The default name is `cluster-client-certificates`.
```

> - **caCertificatesSecret** - Secret name where CA certificate is kept. The default is `nginx-auth-ca`.

Reviewer: ```suggestion
- **caCertificatesSecret** - Name of the Secret which stores the CA certificate. The default name is `nginx-auth-ca`.
```
